coolkitconfig.xcu

```diff
@@ -604,7 +604,7 @@
         </node>
         <node oor:name="CalcText">
             <prop oor:name="Color" oor:op="fuse">
-                <value>15658734</value>
```

Comment:
But I don't really get the second commit. Do they have a known effect, I mean "CalcText" of 15658734 is 0xEEEEEE which is pretty close to 0xFFFFFF already, and "CalcValue" of 7512015 is a bright blue of 0x729FCF and I am unsure where they get used.

Do we need the second commit to fix the known issue?

Reply:
I did not test CalcText and CalcValue how is used, but I changed to be sure all white.

Reply:
Searching in coolkitconfig.xcu I see that the "Light" and "Dark" colors for "CalcText" and "CalcValue" are already quite different from eachother, while "FontColor" is 0 in both. I suggest to drop those two hunks about "Calc*" we're not sure what they affect, seeing as they are already quite different in Dark/Light they are probably ok.

Reply:
Ok, I will drop CalcText and CalcValue, but FontColor it fixes the Writer Text font color case.

Reply:
yeah, it looks odd the have the same "FontColor" in Light and Dark.